dart_io_mini_samples/example/http/handling_an_httprequest_error.dart

```diff
@@ -20,8 +15,12 @@ handleFailure(error) {
   print(error.message);
 }
 
-void main() {
-  http.get("http://some_bogus_website.org")
-    .then(handleSuccess)
-    .catchError(handleFailure);
+main() async {
+  try {
+  	var response = await http.get("http://some_bogus_website.org");
+  	handleSuccess(response);
+  }
+  catch(e) {
```

Comment:
I expected `} catch(e) {` instead of having the `}` on its own line. But use whatever dartfmt produces.